gems/sidekiq/CVE-2023-26141.yml

```diff
@@ -0,0 +1,24 @@
+---
+gem: sidekiq
+cve: 2023-26141
+ghsa: 3qc2-v3hp-6cv8
+url: https://github.com/sidekiq/sidekiq/blob/main/Changes.md#713
```

Comment:
I would just recommend using an NVD or GHSA link to the actual advisory information.

Reply:
You will always see that I use the first public notice from the actual project for  url: if at all possible.